examples/grid.cpp

```diff
@@ -54,9 +55,31 @@ extern "C" {
 #include "common/common_def.h"
 #include "common/condition.h"
 #include "common/lock.h"
+#include "font.h"
 
 using namespace std;
 
+class Timer
```

Comment:
we use this to measure time 
such as 
{
    Timer timer("operation")
    for (int i = 0; i < 30; i++) {
        operation();
    }
}